app/controllers/api/v1/reviews_controller.rb

```diff
@@ -1,22 +1,35 @@
 class Api::V1::ReviewsController < ApplicationController
+
   def create
-    new_review = Review.new(review_params)
     wizard = Wizard.find(review_params[:wizard_id])
     if user_signed_in?
-      # Design descision to be made: will the react element need the wizard?
+      new_review = Review.new(review_params.merge({user_id: current_user.id}))
       if new_review.save
-        render json: { wizard: wizard , review: review_params }
+        render json: { review: review_params }
```

Comment:
you should return the AR review object itself so the react client can receive the newly created id in response